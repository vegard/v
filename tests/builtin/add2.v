x := u64 100;
y := u64 13;

# These should all be the same
print (x + y);
print (_add x y);
print (x._add (y));
