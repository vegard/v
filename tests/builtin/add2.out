113
113
113
